pkg/kms/localkms/localkms_reader.go

```diff
@@ -0,0 +1,49 @@
+/*
+ Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package localkms
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+)
+
+// newReader will create a new local storage storeReader of a keyset with ID value = keysetID
+// it is used internally by local kms
+func newReader(store storage.Store, keysetID string) io.Reader {
```

Comment:
Why not return a struct pointer? It is not needed to return a particular interface from a constructor.

also how will this function be used - i see it is not exported from this package.

Reply:
this will be only used the by the local kms, this is why it's not exported.

the constructor returns a plain io.Reader.. I'll change to storeReader if it makes more sense

Reply:
returning struct pointers makes more sense than returning an interface UNLESS you are trying to meet another interface. (and that's not the case here).

Reply:
sure, I'm replacing it..

Reply:
done